client/src/utils/scroll-to-hash.ts

```diff
@@ -0,0 +1,13 @@
+import {headingIsVisible} from "./heading-is-visible";
+import {getElementTop} from "./get-element-top";
+import {getNavHeight} from "./get-nav-height";
+
+export const scrollToHash = (hash: string): void => {
+  const targets = Array.from(document.querySelectorAll(hash)).filter((e) =>
```

Comment:
We only care about hashes inside of page content right? Do we have to do a document query here?

Reply:
Yea––there might be multiple occurrences of the same ID